pkg/config/ext_authz_test.go

```diff
@@ -254,7 +254,7 @@ func Test_externalAuthZFilter_extAuthz(t *testing.T) {
 			},
 		},
 	}, {
-		name: "http with path prefix",
+		name: "http with pack as bytes option",
```

Comment:
It seems to me that this should be "http with path prefix with pack bytes enabled".